processing/src/main/java/io/druid/query/extraction/ExtractionFn.java

```diff
@@ -40,7 +39,6 @@
     @JsonSubTypes.Type(name = "timeFormat", value = TimeFormatExtractionFn.class),
     @JsonSubTypes.Type(name = "identity", value = IdentityExtractionFn.class),
     @JsonSubTypes.Type(name = "lookup", value = LookupExtractionFn.class),
-    @JsonSubTypes.Type(name = "registeredLookup", value = RegisteredLookupExtractionFn.class),
```

Comment:
why is this removed?

Reply:
I had to move this to https://github.com/a2l007/druid/blob/8f2a360119be5d619a2bc0d57976f483faaad610/server/src/main/java/io/druid/query/lookup/LookupModule.java#L89

as part of the refactoring.